octopus_deploy/tests/conftest.py

```diff
@@ -90,10 +90,15 @@ def method(method, url, file='response', headers=None, params=None):
             param_string = ""
             for key, val in params.items():
                 if type(val) is list:
-                    val_string = ','.join(f'{str(val_item)}' for val_item in val)
+                    if len(val) < 5:
+                        val_string = ','.join(f'{str(val_item)}' for val_item in val)
+                    else:
+                        # don't use full list as the path is too long
+                        val_string = '[]'
                 else:
                     val_string = str(val)
                 param_string += ("/" if param_string else "") + f'{key}={val_string}'
+                param_string = param_string.replace(':', '_')
```

Comment:
```suggestion
                param_string = param_string.replace('_', ':')
```
Shouldn't it be like this? replacing underscores with colons?

Reply:
My bad, I mixed the caddyfiles for fixtures...